util/samplers.py

```diff
@@ -421,5 +475,55 @@ def __call__(self, score_model, sampled_energies, init_x, batch_size=1):
                 
         # Do not include noise in last step
         #x_mean = x_mean
+        if not diffusion_on_mask:
+          x = x * mask_tensor
         return x
-    
\ No newline at end of file
+    
```

Comment:
These were removed as separate functions and added as class methods above. We don't need duplicates here, you should just call the class method now rather than the functions

Reply:
Done